packages/@react-stately/data/src/useAsyncList.ts

```diff
@@ -230,6 +234,12 @@ function reducer<T, C>(data: AsyncListState<T, C>, action: Action<T, C>): AsyncL
             items: action.type === 'loading' ? [] : data.items,
             abortController: action.abortController
           };
+        case 'loadingMore':
+          // If already loading more and another loading more is triggered, abort the new load more.
```

Comment:
Can you just add a little bit more about "the cursor hasn't been updated so they are duplicate requests"?